eloquentjs/2_1_looping_a_triangle.js

```diff
@@ -0,0 +1,61 @@
+/**Draws a right triangle utilizing the += operator*/
+function drawTriangle(height) { 
+    if (isNaN(height)) {
+        throw "Triangle height cannot be NaN";
+        return;
+    }
+    if (height <= 0) {
+        throw "Triangle height cannot be zero or less";
+        return;
+    } 
+    let tri = '';
+    for (let i = 0; i <= height; i++) {
+        for (let j = 0; j < i; j++) {
+            tri += "#";
+        }
+        tri += "\n";
+    }
+    console.log(tri);
+}
+
+/**Draws a right triangle utilizing array join method*/
```

Comment:
You should always have a space between the comment chars and the comment body. In the case of a docstring, the standard is to use new lines.